packages/database/src/migrations/migrationData/20200826215113-UseIndicatorsInStriveVisualisations.js

```diff
@@ -22,8 +22,8 @@ export const indicators = {
     config: {
       formula: 'MRDT_POS_CONSULT_WEEKLY / SSWT1072',
       aggregation: {
-        MRDT_POS_CONSULT_WEEKLY: 'SUM',
-        SSWT1072: ['FINAL_EACH_WEEK', 'SUM'],
+        MRDT_POS_CONSULT_WEEKLY: 'SUM_UNTIL_THIS_DAY',
+        SSWT1072: ['FINAL_EACH_WEEK', 'SUM_UNTIL_THIS_DAY'],
```

Comment:
This migration has not run in `dev`, so it's simpler to update it and re-deploy the feature instance

Reply:
Makes sense.